src/Runner.Worker/JobExtension.cs

```diff
@@ -239,7 +239,18 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                     // Download actions not already in the cache
                     Trace.Info("Downloading actions");
                     var actionManager = HostContext.GetService<IActionManager>();
-                    var prepareResult = await actionManager.PrepareActionsAsync(context, message.Steps);
+                    PrepareResult prepareResult;
```

Comment:
can we make an actionable error if we're connected to an old server and can't download